e2e-tests/insurance/cypress/e2e/journeys/policy/multiple-contract-policy/validation/multiple-contract-policy-validation-total-months-of-cover.spec.js

```diff
@@ -89,6 +89,18 @@ context('Insurance - Policy - Multiple contract policy page - form validation -
     });
   });
 
+  describe('when total months of cover contains a decimal', () => {
+    it('should render a validation error', () => {
+      cy.submitAndAssertFieldErrors(
+        field,
+        '7.5',
```

Comment:
constant?

Reply:
What kind of a constant? I don't think we shouldn't constants for plain numbers?

Reply:
for example, I have no idea why it's 7.5.  but like const MONTH_WITH_DECIMAL would explain better?

Reply:
Is the test description not enough? "when total months of cover contains a decimal"

We have decimal tests like this everwhere with hard coded decimal values, e.g search for "contains a decimal"

Thing is, 7.5 is not month specific, it's just a number and so if we want this (and values with decimals) to be a constants, I suppose they should be generic, in which case how would a number constant be named?